controllers/cloud.redhat.com/providers/database/appinterface.go

```diff
@@ -60,15 +60,26 @@ func NewAppInterfaceDBProvider(p *providers.Provider) (providers.ClowderProvider
 }
 
 func (a *appInterface) EnvProvide() error {
-	if rdsCa == "" {
-		_rdsCa, err := fetchCa()
+	caURL := a.Env.Spec.Providers.Database.CaBundleURL
+	if caURL == "" {
+		caURL = defaultCaBundleURL
+	}
+
+	_, ok := rdsCaBundles[caURL]
+	if !ok {
+		rdsCaBundles[caURL] = ""
+	}
+
+	if rdsCaBundles[caURL] == "" {
```

Comment:
why not put this inside the above conditional?

Reply:
I am checking for 2 separate conditions:
1. the key may not yet exist at all
2. the key may exist, but be and empty string because the last time an HTTP GET was attempted it failed

Reply:
OK no need to check for the key presence, I can just combine this into 1 'if' ... `if rdsCaBundles[caURL] == "" {`